templates/overrides/navigation/pager.html.twig

```diff
@@ -1,9 +1,24 @@
 {#
 /**
  * @file
- * Template for a pager.
+ * Override for core 'pager' template, using the 'navigation' pattern.
  */
 #}
-{% include '@oe_bootstrap_theme/overrides/pagination/pagination.html.twig' with {
```

Comment:
we also need to remove this file '@oe_bootstrap_theme/overrides/pagination/pagination.html.twig'